packages/legend-manual-tests/src/__tests__/roundtrip-grammar/RoundtripGrammar.test.ts

```diff
@@ -89,6 +89,9 @@ const EXCLUSIONS: { [key: string]: ROUNTRIP_TEST_PHASES[] | typeof SKIP } = {
   'nested-embedded-relational-mapping.pure': SKIP,
   'relational-mapping-filter.pure': SKIP,
 
+  // TODO: remove `SKIP` flag when compilation is supported in engine
+  'username-password-auth-connection.pure': SKIP,
```

Comment:
I think Pierre just merged a bunch of things on engine side, could we try to remove this skip now?